language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/extensions/document/SyntaxApiQuoteTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.langserver.extensions.document;
+
+import org.ballerinalang.langserver.extensions.LSExtensionTestUtil;
+import org.ballerinalang.langserver.extensions.ballerina.document.BallerinaSyntaxApiQuoteResponse;
+import org.ballerinalang.langserver.util.FileUtils;
+import org.ballerinalang.langserver.util.TestUtil;
+import org.eclipse.lsp4j.jsonrpc.Endpoint;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Test SyntaxApiQuote methods.
+ *
+ * @since 2.0.0
+ */
+public class SyntaxApiQuoteTest {
+    private final Path emptyFile = FileUtils.RES_DIR.resolve("extensions")
```

Comment:
Get `FileUtils.RES_DIR.resolve("extensions").resolve("document").resolve("syntaxApiQuote")` to a variable and reuse.